src/annotator/cross-frame.js

```diff
@@ -55,70 +56,76 @@ export class CrossFrame {
     };
 
     const iframeUnloaded = frame => {
-      bridge.call('destroyFrame', frameIdentifiers.get(frame));
+      this._bridge.call('destroyFrame', frameIdentifiers.get(frame));
       frameIdentifiers.delete(frame);
     };
 
-    frameObserver.observe(injectIntoFrame, iframeUnloaded);
+    this._frameObserver.observe(injectIntoFrame, iframeUnloaded);
+  }
 
-    /**
-     * Attempt to connect to the sidebar frame.
-     *
-     * Returns a promise that resolves once the connection has been established.
-     *
-     * @param {Window} frame - The window containing the sidebar application
-     * @param {string} origin - Origin of the sidebar application (eg. 'https://hypothes.is/')
-     */
-    this.connectToSidebar = (frame, origin) => {
-      const channel = new MessageChannel();
-      frame.postMessage(
-        {
-          type: 'hypothesisGuestReady',
-        },
-        origin,
-        [channel.port2]
-      );
-      bridge.createChannel(channel.port1);
-    };
+  /**
+   * Attempt to connect to the sidebar frame.
+   *
+   * Returns a promise that resolves once the connection has been established.
+   *
+   * @param {Window} frame - The window containing the sidebar application
+   * @param {string} origin - Origin of the sidebar application (eg. 'https://hypothes.is/')
+   */
+  connectToSidebar(frame, origin) {
+    const channel = new MessageChannel();
+    frame.postMessage(
+      {
+        type: 'hypothesisGuestReady',
+      },
+      origin,
+      [channel.port2]
+    );
+    this._bridge.createChannel(channel.port1);
+  }
 
-    /**
-     * Remove the connection between the sidebar and annotator.
-     */
-    this.destroy = () => {
-      bridge.destroy();
-      annotationSync.destroy();
-      frameObserver.disconnect();
-    };
+  /**
+   * Remove the connection between the sidebar and annotator.
+   */
+  destroy() {
+    this._bridge.destroy();
+    this._annotationSync.destroy();
+    this._frameObserver.disconnect();
+  }
 
-    /**
-     * Notify the sidebar about new annotations created in the page.
-     *
-     * @param {AnnotationData[]} annotations
-     */
-    this.sync = annotations => annotationSync.sync(annotations);
+  /**
+   * Notify the sidebar about new annotations created in the page.
+   *
+   * @param {AnnotationData[]} annotations
+   */
+  sync(annotations) {
+    this._annotationSync.sync(annotations);
+  }
 
-    /**
-     * Subscribe to an event from the sidebar.
-     *
-     * @param {string} event
-     * @param {(...args: any[]) => void} callback
-     */
-    this.on = (event, callback) => bridge.on(event, callback);
+  /**
+   * Subscribe to an event from the sidebar.
+   *
+   * @see {Bridge.on} for details.
+   */
```

Comment:
Ah there was a misunderstanding here. I meant to suggest to leave the `@param {type} name` comments in place, but remove the explanatory comments.

Reply:
I mended to include only the `@param` and the `@see` for more explanation